core/collector.class.inc.php

```diff
@@ -594,7 +594,7 @@ protected function AddRow($aRow)
 	{
 		$aData = array();
 		foreach ($this->aCSVHeaders as $sHeader) {
-			if (is_null($aRow[$sHeader]) && $this->AttributeIsNullified($sHeader)) {
+			if (strlen($aRow[$sHeader] ?? '')===0 && $this->AttributeIsNullified($sHeader)) {
```

Comment:
Empty string is not the same as `null`.

Reply:
No, The whole thing is to be able to set a field's value as `NULL` in the SQL database, not reset, not ignore!

So it seems it will still not work and Combodo still doesn't understand the needs.. 😢